reliability/setup.yml

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
@jeremyeder @vikaslaad  Here is a link to the playbook we have - https://github.com/distributed-system-analysis/pbench/tree/master/contrib/ansible/openshift. The labeling of the nodes starts from 0 instead of 1, I have a PR which fixes that - https://github.com/distributed-system-analysis/pbench/pull/551.